src/coreclr/tools/Common/JitInterface/CorInfoTypes.cs

```diff
@@ -1152,6 +1152,8 @@ public unsafe struct CORINFO_THREAD_STATIC_BLOCKS_INFO
     {
         public CORINFO_CONST_LOOKUP tlsIndex;
         public uint offsetOfThreadLocalStoragePointer;
+        public ulong tlsGetAddrFtnPtr;
+        public ulong descrAddrOfMaxThreadStaticBlock;
```

Comment:
```suggestion
        public unint tlsGetAddrFtnPtr;
        public unint descrAddrOfMaxThreadStaticBlock;
```
unint is the correct equivalent for `size_t` in C++